proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,258 @@
+# Your Proposal Title
+
+* **Owners:**
+  * Daniel Schürmann
+
+* **Implementation Status:** `Not implemented`
```

Comment:
we should have a bullet for core members to officially approve the document (I'd prefer to have it inline rather than just as a PR approval). We can each file a separate signed commit to add our names

```suggestion
* **Implementation Status:** `Not implemented`

* **Signed By:**
```

Reply:
The German version of this Satzung needs to be signed by at least 7 founding members of Mixxx e.V. I need to check how this can be done electronically. Not sure if a self created certificate is valid for this.

Reply:
Let's make it easy and invest in 7 post stamps.